airbyte-integrations/connectors/source-harvest/source_harvest/schemas/time_team.json

```diff
@@ -0,0 +1,27 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "properties": {
+    "user_id": {
+      "type": ["null", "integer"]
+    },
+    "user_name": {
+      "type": ["null", "string"]
+    },
+    "is_contractor": {
+      "type": ["null", "boolean"]
+    },
+    "total_hours": {
+      "type": ["null", "float"]
```

Comment:
I'm not sure if there is such a datatype for the field of the schema. Could you check this moment please?